packages/cadl-ranch-specs/http/models/bodycomplex/cadl-output/cadl/testserver/bodycomplex/aio/_client.py

```diff
@@ -0,0 +1,109 @@
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import Any, Awaitable
+
+from azure.core import AsyncPipelineClient
+from azure.core.rest import AsyncHttpResponse, HttpRequest
+
+from .. import models
+from .._serialization import Deserializer, Serializer
+from ._configuration import ComplexTestServiceConfiguration
+from .operations import (
+    ArrayOperations,
+    BasicOpsOperations,
+    DictionaryOperations,
+    FlattencomplexOperations,
+    InheritanceOperations,
+    PolymorphicrecursiveOperations,
+    PolymorphismOperations,
+    PrimitiveOperations,
+    ReadonlypropertyOperations,
+)
+
+
+class ComplexTestService:  # pylint: disable=client-accepts-api-version-keyword,too-many-instance-attributes
+    """Service client.
+
+    :ivar basic_ops: BasicOpsOperations operations
+    :vartype basic_ops: cadl.testserver.bodycomplex.aio.operations.BasicOpsOperations
+    :ivar primitive: PrimitiveOperations operations
+    :vartype primitive: cadl.testserver.bodycomplex.aio.operations.PrimitiveOperations
+    :ivar array: ArrayOperations operations
+    :vartype array: cadl.testserver.bodycomplex.aio.operations.ArrayOperations
+    :ivar dictionary: DictionaryOperations operations
+    :vartype dictionary: cadl.testserver.bodycomplex.aio.operations.DictionaryOperations
+    :ivar inheritance: InheritanceOperations operations
+    :vartype inheritance: cadl.testserver.bodycomplex.aio.operations.InheritanceOperations
+    :ivar polymorphism: PolymorphismOperations operations
+    :vartype polymorphism: cadl.testserver.bodycomplex.aio.operations.PolymorphismOperations
+    :ivar polymorphicrecursive: PolymorphicrecursiveOperations operations
+    :vartype polymorphicrecursive:
+     cadl.testserver.bodycomplex.aio.operations.PolymorphicrecursiveOperations
+    :ivar readonlyproperty: ReadonlypropertyOperations operations
+    :vartype readonlyproperty:
+     cadl.testserver.bodycomplex.aio.operations.ReadonlypropertyOperations
+    :ivar flattencomplex: FlattencomplexOperations operations
+    :vartype flattencomplex: cadl.testserver.bodycomplex.aio.operations.FlattencomplexOperations
+    """
+
+    def __init__(self, **kwargs: Any) -> None:  # pylint: disable=missing-client-constructor-parameter-credential
+        _endpoint = "http://localhost:3000"
+        self._config = ComplexTestServiceConfiguration(**kwargs)
+        self._client = AsyncPipelineClient(base_url=_endpoint, config=self._config, **kwargs)
+
+        client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
+        self._serialize = Serializer(client_models)
+        self._deserialize = Deserializer(client_models)
```

Comment:
It is not used for `dpg model`, So shall we remove it?

Reply:
removed.